### PR TITLE
Completed Navigation Component

### PR DESCRIPTION
**Pull request by GarvG**
Completed My Navigation Component for CL 
files to be Checked
Navigation.html 
Navigation.css

### CODE REVIEW

File: Navigation.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+}
+.nav{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    
+    width: 55rem;
+   
+
```
Comment (srishtigautam21) [inline Navigation.css:16]: Give consistent spacing

File: Navigation.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+.Nav_Links
+{
+    list-style: none;
+
+}
+.Nav_Links .Nav_links_li
+{
+    display: inline-block;
+    padding: 0px 1.3rem;
+}
```
Comment (srishtigautam21) [inline Navigation.css:31]: You can remove px when value is zero

**Review by srishtigautam21 — APPROVED** [review]
The code looks clean
Good to merge

File: Navigation.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <!-- using the below link for the google fonts -->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <!-- Using the Below Link for the awesome icons -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (iRohitGaur) [inline Navigation.html:12]: appreciation: good use of comments

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: Navigation.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
+@import url(components.css);
+.Nav_links_li, .nav_links_a, button{
```
Comment (amartya-rs) [inline Navigation.css:2]: suggestion: maintain consistency in class names. Some names are starting with capitals while some with small.

**Review by amartya-rs — APPROVED** [review]
Good to merge with the suggested changes.

File: Navigation.css (hunk 1 above)
Comment (GarvG) [reply Navigation.css:16]: will take care from next time! thank you for the Suggestion

File: Navigation.css (hunk 2 above)
Comment (GarvG) [reply Navigation.css:31]: Done

File: Navigation.html (hunk 3 above)
Comment (GarvG) [reply Navigation.html:12]: Thank you for the appreciation!

File: Navigation.css (hunk 4 above)
Comment (GarvG) [reply Navigation.css:2]: Will take Care from the next time, thank you for the suggestions

File: Navigation.css [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    font-size: 1rem;
+    color:var(--textgray);
+    text-decoration: none;
+
+}
+.nav{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
```
Comment (iamalhera) [inline Navigation.css:13]: avoid extra indentation

**Pull request merged**